R/gi_available_templates.R

```diff
@@ -12,7 +12,7 @@
 gi_available_templates <-
   function() {
 
-    url = "https://www.gitignore.io/api/list?format=json"
+    url = "https://www.toptal.com/developers/gitignore/api/list?format=json"
```

Comment:
Nit: Consider hiding the actual URL behind an interface so next time the backend changes you need to update it in just one place.

```r
backend_url <- function() {
  "https://www.toptal.com/developers/gitignore"
}
```

Line 15 then becomes

```r
    url = glue::glue("{backend_url()}/api/list?format=json}")
```